### DISCUSSION TITLE
[Enhancement] MAUI Font Icon

### DISCUSSION THREAD

**Discussion by WebAssemblyMan** [dotnet/maui#177] [General]
## MAUI Font Icon

We have the following font icons that are tied to the look and feel of the different native platforms:

Microsoft - Segoe MDL2 Icon
Google - Material Design Icons 
Apple - SF Symbols

We also have popular font icons such as fontawesome and openiconic who provide a free version but up-sell a pro version. I hope the MAUI team can provide a basic set of icons that works beautifully across the different platforms for MAUI. Something like ionicicons. Something designed for cross platform. This will help small developers like me to get started quickly in creating good UI.

**Comment by Happypig375** [comment]
You don't need to bold the two false statements.

**Comment by WebAssemblyMan** [comment]
True? False? definitely classic 😁

**Comment by Happypig375** [comment]
So you call your made-up false statement "classic"?
